README.md

```diff
@@ -71,6 +72,23 @@ Or, if you've been mimicked a little harder,
 <img alt="some worse code"
      src="https://cloud.githubusercontent.com/assets/1236420/10564914/f7963ae4-7591-11e5-9b45-f123e42b22f4.png" />
 
+### Installation
```

Comment:
I thought it more natural for the installation to be described before usage - as such, this section would need to be moved up a little.